inc_internal/message.h

```diff
@@ -90,7 +90,7 @@ bool message_get_int32_header(message *m, int header_id, int32_t *v);
 
 bool message_get_uint64_header(message *m, int header_id, uint64_t *v);
 
-bool message_get_bytes_header(message *m, int header_id, const char **ptr, size_t *len);
+bool message_get_bytes_header(message *m, int header_id, uint8_t **ptr, size_t *len);
```

Comment:
that is a bad idea -- this returns the pointer to data owned by the message -- therefore marked const to the caller.
The caller cannot assume that the data is modifiable.

Reply:
changed signature back to const.